op-dispute-mon/mon/extract/extractor.go

```diff
@@ -54,11 +54,17 @@ func (e *Extractor) enrichGames(ctx context.Context, games []gameTypes.GameMetad
 			e.logger.Error("failed to fetch game claims", "err", err)
 			continue
 		}
+		gameDuration, err := caller.GetGameDuration(ctx)
```

Comment:
We should fetch this as part of `GetGameMetadata` given that's a custom batch call to grab the data we need for this monitoring.